bluetooth_mesh/cli/gatt_client.py

```diff
@@ -100,7 +102,27 @@ def _mesh_beacon_received(self, _, payload):
             self._network.beacon_receive(beacon, auth)
 
     def _network_pdu_received(self, _, payload):
-        pass
+        iv_index, ctl, ttl, seq, src, dst, transport_pdu= NetworkMessage.unpack(payload, 
+                                        self._network.network_keys, self._network.iv_index)
+        if transport_pdu:
+            if src >= 0x8000 or dst == 0:
+                return
+            for node in self._network.nodes.values():
+                for ele in node.elements:
+                    if ele['unicastAddress'] == src:
+                        if (iv_index << 24 | seq) <= (self._network.iv_index << 24 | ele["sequenceNumber"]):
+                            return
+                        ele["sequenceNumber"] = seq
```

Comment:
If this part of the network JSON is to be used, I'd rather see a proper `Element` class, instead of using dictionaries returned by the schema.

Reply:
one question, how we handle the dynamic information, stored in the json database or in the cache?
and i agree that an Element class helps to extract information from schema and more OO.